src/lib/FreeC/Backend/Coq/Base.hs

```diff
@@ -56,6 +56,10 @@ freeArgs =
 -- Partiality                                                                --
 -------------------------------------------------------------------------------
 
+-- | The name of the type class @Partial@
```

Comment:
```suggestion
-- | The Coq identifier for the @Partial@ type class.
```
I would change the wording here slightly to be more in line with the rest of the comments.